packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -78,24 +79,35 @@ export default async args => {
     }
 
     if (platform === "native") {
-        result.push({
-            input: widgetEntry,
-            output: {
-                format: "es",
-                file: join(outDir, outWidgetFile),
-                sourcemap: false
-            },
-            external: nativeExternal,
-            plugins: [
-                ...getClientComponentPlugins(),
-                json(),
-                ...getCommonPlugins({
-                    sourceMaps: false,
-                    extensions: nativeExtensions,
-                    transpile: false
-                })
-            ],
-            onwarn
+        ["ios", "android"].forEach((os, i) => {
```

Comment:
I think we need to extend our integration tests to cover such native widget. Including a check that `json` file is generated correctly and `node_modules` are copied

Reply:
For this to happen we need to add a lib containing native dependency in the templates

Reply:
I added an alternative that tests the results (json, node_modules and transitives node_modules)